src/main/java/frc/robot/RobotContainer.java

```diff
@@ -43,35 +45,35 @@ public class RobotContainer {
 
     private CANSparkMax backRightAngleMotor = new CANSparkMax(DrivetrainConstants.BACK_RIGHT_ANGLE_ID, MotorType.kBrushless);
     private CANSparkMax backRightSpeedMotor = new CANSparkMax(DrivetrainConstants.BACK_RIGHT_SPEED_ID, MotorType.kBrushless);
-    private AnalogEncoder backRightEncoder = new AnalogEncoder(DrivetrainConstants.BACK_RIGHT_ENCODER);
+    // private AnalogEncoder backRightEncoder = new AnalogEncoder(DrivetrainConstants.BACK_RIGHT_ENCODER);
```

Comment:
What do you mean duplicate? Is it bad to have any commented encoders? Right now I've just commented the encoders because they're not used in the max swerve subsystem

Reply:
My bad, misread the code, ignore